source/features/forked-to.tsx

```diff
@@ -49,26 +50,22 @@ async function updateUI(forks: string[]): Promise<void> {
 	await elementReady('.page-actions');
 
 	const forkButton = select('.pagehead-actions [aria-label^="Fork your own copy of"]')!;
-	forkButton.classList.add('rounded-left-2', 'BtnGroup-item', 'mr-0');
-
-	if (forks.length === 1) {
-		forkButton.after(
+	const forkedToButton = forks.length === 1
+		? (
 			<a
 				href={createLink(forks[0])}
-				className="btn btn-sm BtnGroup-item px-2 rgh-forked-button rgh-forked-link"
+				className="btn btn-sm px-2  rgh-forked-link"
```

Comment:
```suggestion
				className="btn btn-sm px-2 rgh-forked-link"
```

Reply:
I fixed it manually because I didn't see the commit suggestion button earlier, sorry but thank you for the suggestion

Reply:
No problem, I don't have to be credited for a one whitespace character fix :wink: